app/assets/javascripts/experiments/table.js

```diff
@@ -42,6 +42,16 @@ var ExperimnetTable = {
       $(`<div class="table-row" data-id="${id}">${row}</div>`).appendTo(`${this.table} .table-body`);
     });
   },
+  initArchiveMyModules: function() {
+    $('#archiveTask').on('click', (e) => {
```

Comment:
Shouldn't we use an event delegation for such actions?

Reply:
Toolbar static, we render it on load once. I think we can assign an action to the button directly.